pages/ActivityStreams/Overview.md

```diff
@@ -79,13 +81,15 @@ And implementers may choose to ignore the following link objects:
 #### Note
 
 Note objects must be supported to represent text content posted by users.
-Note objects must have a `"type"` field with a value of `"Note"` and a `"content"` field with a string.
-Clients should support content with unicode text as possible in their given browser or operating system.
+Note objects must include a `"type"` field with a value of `"Note"`.
+Note objects must include a `"content"` field with a string.
+
+Clients may support content with unicode text as possible in their given browser or operating system.
 Clients may support [Markdown](https://daringfireball.net/projects/markdown/), [BBCode](https://www.bbcode.org) or other formatting standards.
-If a note contains Markdown, BBCode or some other formatting, it is recommended that a `"mediaType"` field be added to the object with a [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) string representing the type of encoding present, i.e. `"text/markdown"`.
+If a note contains Markdown, BBCode or some other formatting, the note object should include a `"mediaType"` field with a [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) string representing the type of encoding present, i.e. `"text/markdown"`.
```

Comment:
Under what conditions would a note that isn't text/plain *not* need a mimetype specified?  Maybe this should be required?

Do we want to limit notes to text?  "If specified, the mediaType field must contain a media type with a top level of 'text'"

Reply:
Changed to must